src/revChatGPT/__main__.py

```diff
@@ -118,58 +117,26 @@ def main():
                     continue
                 elif prompt == "!refresh":
                     chatbot.refresh_session()
-                    print("Session refreshed.\n")
+                    print("Session refreshed.")
                     continue
                 elif prompt == "!rollback":
-                    chatbot.rollback_conversation()
-                    print("Chat session rolled back.")
+                    chatbot.rollback()
```

Comment:
This is the wrong call. 

https://github.com/acheong08/ChatGPT/blob/5903565d75a180a0693c6f8ddcb48917c18cd069/src/revChatGPT/revChatGPT.py#L241-L249